frontend/packages/dev-console/src/components/topology/list-view/TopologyListViewNode.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import {
+  DataListCell,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+} from '@patternfly/react-core';
+import { isNode, Node, observer } from '@patternfly/react-topology';
+import { useSearchFilter } from '../filters';
+import { getResourceKind } from '../topology-utils';
+import { labelForNodeKind } from './list-view-utils';
+import {
+  AlertsCell,
+  GroupResourcesCell,
+  MetricsCell,
+  StatusCell,
+  TypedResourceBadgeCell,
+} from './cells';
+// eslint-disable-next-line import/no-cycle
+import { TopologyListViewNodeChildren } from './TopologyListViewNodeChildren';
```

Comment:
Uh... that can't be good. This can shoot us in the foot if the import resolutions change order. If it hits a circular reference on first import than it resolves as undefined (I'm pretty sure). Otherwise it caches it and we get lucky. Is there not a better way to manage the imports?